bin/lib/versions.js

```diff
@@ -85,17 +85,59 @@ exports.get_apple_xcode_version = function() {
     child_process.exec('xcodebuild -version', function(error, stdout, stderr) {
         if (error) {
             d.reject(stderr);
+        } else {
+            var version = stdout.split('\n')[0].slice(6);
```

Comment:
Updated in https://github.com/MSOpenTech/cordova-ios/commit/079fcf3c32aa8fce98ad68ee062aaff00cb0dab3